walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,86 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    // throw new UnsupportedOperationException("TODO: Implement this method.");
+
+    //  get a list of individuals we need to consider from request
+    Collection<String> attendees = request.getAttendees();
+    long duration = request.getDuration();
+    if (duration > TimeRange.END_OF_DAY - TimeRange.START_OF_DAY) {
+        return new ArrayList<TimeRange>();
+    }
+    
+    // isolate the timeranges from events that pertain to those individuals
+    ArrayList<TimeRange> timeRanges = new ArrayList<TimeRange>();
```

Comment:
`timeRanges` is too ambiguous of a variable name. Maybe something like `busyTimeRanges`?